clouddriver-ecs/src/integration/resources/clouddriver.yml

```diff
@@ -24,10 +24,16 @@ aws:
 ecs:
   enabled: true
   primaryAccount: ecs-account
+  defaultNamingStrategy: tags
```

Comment:
There's two naming strategies included in this PR `default` (see `ECSDefaultNamer`) and `tags` (see `ECSTagNamer`). If there's no value for `defaultNamingStrategy` it will have the value of `default` which is set at a field level in `ECSCredentialsConfig`.  This will then be used in the `EcsCredentialsParser` naming strategy lookup.

The lookup will check for a non-blank account level value in `namingStrategy` and then use the value from the provider level `defaultNamingStrategy`

If the value for `defaultNamingStrategy` or `namingStrategy` are blank or otherwise invalid (i.e. don't map to a registered `Namer`) then the account parsing will fail with the following exception:

```Caused by: java.lang.IllegalArgumentException: Could not find naming strategy 'invalid'
	at com.netflix.spinnaker.clouddriver.names.NamerRegistry.lambda$getNamingStrategy$1(NamerRegistry.java:58)
	at java.base/java.util.Optional.orElseThrow(Optional.java:408)
	at com.netflix.spinnaker.clouddriver.names.NamerRegistry.getNamingStrategy(NamerRegistry.java:56)
	at com.netflix.spinnaker.clouddriver.ecs.security.EcsCredentialsParser.parse(EcsCredentialsParser.java:70)
```